PREPEND-README.md

```diff
@@ -0,0 +1,42 @@
+# Bash hook to prepend branch name
+This explains how to use and setup a git hook that will prepend the issue number
+to a commit message. Git hooks are used to automate git commands and functions. There are a number of different options, but this particular hook will add the issue number to your "git commit-m" message once it is pushed to the repository. Git hooks live in a local git/hooks file of each repo, so initializing this hook in the robotics-prototype repo will not change any other repos you might have.
+
+## Setting up a git hook
+Windows:
+In your space concordia directory find the git folder and open hooks contained there in. It should look like this:
+
+robotics-protoype/git/hooks
+
+once there open prepare-commit-msg.sample in your editor. Replace the code in the sample with the code found in commit-message-hook.sh. Save the file by keeping the name the same and removing the .sample suffix.
+
+Linux:
```

Comment:
I think these instructions can be simplified, here is my suggestion:

From the root of the repository:
- `cp commit-message-hook.sh .git/hooks/prepare-commit-msg`

That command will copy it over to the right directory and rename it properly, while keeping the sample file intact.

Additionally you should `chmod +x` the script itself and commit the change so that it is executable by default.